.travis.yml

```diff
@@ -39,6 +39,24 @@ matrix:
       after_failure:
           -  cat `find out/junit-results -name "*.txt" -exec grep -l FAILED {} \;`
 
+    - name: "c: linux gcc 8 Wextra"
```

Comment:
PR says gcc 9, this says gcc 8. please fix one or the other.

Reply:
oh, i see you had a commit where you downgraded from 9 to 8. i will edit the pr.